sample/app/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,72 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.rokolabs.mparticle.sample">
+
+    <application
+        android:name=".MParticleSampleApplication"
+        android:allowBackup="true"
+        android:icon="@mipmap/ic_launcher"
+        android:label="@string/app_name"
+        android:roundIcon="@mipmap/ic_launcher_round"
+        android:supportsRtl="true"
+        android:theme="@style/AppTheme">
+        <activity
+            android:name=".MainActivity"
+            android:label="@string/app_name"
+            android:theme="@style/AppTheme.NoActionBar">
+            <intent-filter>
+                <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
+            </intent-filter>
+        </activity>
+
+        <activity android:label="Events"  android:name=".EventsActivity" >
+            <intent-filter>
+                <action android:name="android.intent.action.VIEW" />
+                <category android:name="android.intent.category.DEFAULT" />
+                <category android:name="android.intent.category.BROWSABLE" />
+                <data
+                    android:host="rokomparticle.rkm.io"
+                    android:pathPrefix="/"
+                    android:scheme="https" />
+            </intent-filter>
+        </activity>
+        <activity android:label="Users" android:name=".UserActivity" />
+
+        <activity android:label="App page"  android:name=".AppPageActivity">
+            <intent-filter>
+                <action android:name="com.rokolabs.mobi.PUSH_ACTION" />
+                <category android:name="android.intent.category.DEFAULT" />
+            </intent-filter>
+        </activity>
+
+        <activity android:label="Instabot" android:name=".InstabotUIActivity">
+            <intent-filter>
+                <action android:name="com.rokolabs.mobi.INSTABOT_ACTIVITY" />
+                <category android:name="android.intent.category.DEFAULT" />
+            </intent-filter>
+        </activity>
+
+        <receiver android:name="com.mparticle.MPReceiver"
+            android:permission="com.google.android.c2dm.permission.SEND">
+            <intent-filter>
+                <action android:name="com.google.android.c2dm.intent.RECEIVE" />
+                <!-- Use your package name as the category -->
+                <category android:name="com.mparticle.kits.pushsample" />
+            </intent-filter>
+        </receiver>
+
+        <service android:name="com.mparticle.MPService" />
+
+        <receiver android:name="com.mparticle.ReferrerReceiver" android:exported="true">
+            <intent-filter>
+                <action android:name="com.android.vending.INSTALL_REFERRER"/>
+            </intent-filter>
+        </receiver>
+
+        <meta-data
+            android:name="ROKOMobiAPIToken"
+            android:value="uin9WU4YYh5Xqm1N+8B9fP8wonMXjTpD0S24rXt2Cp4=" />
```

Comment:
there should be no reason to have the API token in the app. If that's necessary, then there's some issue with the integration. All settings should be server-side.

Reply:
agreed. we'll update PR to remove this key from AndroidManifest.xml, and add new method to pass in the Instabot API key for SDK initialization from the mParticle wrapper so it can be set server-side.